requirements/dev.txt

```diff
@@ -3,3 +3,4 @@ pika>=1.0.0
 kombu==4.6.*
 ujson==3.0.0
 django-model-utils==4.0.0
+python-dateutil >= 2.4
```

Comment:
Could you explain, why did you add this library?

Reply:
It's used for parsing expires string (from_message) to datetime with timezone info.
In python 3.6 i didn't found good solution,  datetime.fromisoformat only for python >= 3.7.
At summary:
- dateutil already at connect dependencies
- it's pretty stable
- 2.4 released at 2015 and should't broke any library dependencies

Reply:
Okay, let's leave.
I just thought it was there in Django: 
https://docs.djangoproject.com/en/2.2/_modules/django/utils/dateparse/

But I don't know about version 3.